Dockerfile

```diff
@@ -1,24 +1,24 @@
-FROM ubuntu:16.04
+FROM python:2-slim
 
+WORKDIR /app
 ARG DEBIAN_FRONTEND=noninteractive
 ARG PIP_NO_CACHE_DIR=0
+ENTRYPOINT ["make"]
```

Comment:
I understand that you reordered the Dockerfile commands to achieve (somewhat) better layer caching when re-building the Docker image, but could you please add a bit of whitespace or a comment here?

In the sense that similar commands should be grouped together, but different one shouldn't.

Reply:
Separated :)